release notes/v0.47.0.md

```diff
@@ -0,0 +1,113 @@
+- [#3335](https://github.com/grafana/k6/pull/3335) The character limit for metric names increased from 63 to 128 after the OpenTelemetry update. k6 will return an error starting on the next release (v0.48.0) if users hit the limit.
+- [browser#1007](https://github.com/grafana/xk6-browser/pull/1007) Adds a `k6` object (`window.k6 = {};`) to help identify k6 browser module tests.
+- [browser#1022](https://github.com/grafana/xk6-browser/pull/1022) Refactors the `check` in `examples/fillform.js` so that it matches the type definitions and documentation for `check`.
+
+## Bug fixes
+
+- [xk6-grpc#47](https://github.com/grafana/xk6-grpc/pull/47) Fixes the premature closing of a gRPC stream when a stream's client has finished sending. Thanks to @thiagodpf for reporting!
+- [#3344](https://github.com/grafana/k6/pull/3344), [xk6-grpc#49](https://github.com/grafana/xk6-grpc/pull/49) Adds support for Google's protobuf wrappers. Thanks to @zibul444 for reporting!
+- [#3346](https://github.com/grafana/k6/pull/3346) Fixes a race condition when `http.batch` is used with a binary response.
+- [#3308](https://github.com/grafana/k6/pull/3308) Updates `goja` version, and fixes a compiler bug when a class is declared in a function with an argument.
+- [browser#1040](https://github.com/grafana/xk6-browser/pull/1040) Fixes the `context.clearCookies()` method to clear all cookies from the current browser context.
+- [browser#1031](https://github.com/grafana/xk6-browser/pull/1031) Fixes the `expires` field while adding cookies using the `context.addCookie()` method.
+- [browser#1039](https://github.com/grafana/xk6-browser/pull/1039) Fixes `goja` conversions while adding and retrieving cookies.
+- [browser#1038](https://github.com/grafana/xk6-browser/pull/1038) Fixes read/write data race for edge case with remote browsers.
+- [browser#1035](https://github.com/grafana/xk6-browser/pull/1035) Refactors `int64` timeout to `time.Duration`, to help avoid confusion as to whether a timeout is in milliseconds or seconds.
+- [browser#1034](https://github.com/grafana/xk6-browser/pull/1034) Fixes `page.reload` & `page.setContent` to use the default navigation timeout over the default timeout.
+- [browser#1033](https://github.com/grafana/xk6-browser/pull/1033) Fixes the `page` timeouts so it is actually used after being set.
+
+## Maintenance and internal improvements
+
+- [#3342](https://github.com/grafana/k6/pull/3342) Updates xk6-grpc to the latest version. This change brings all the latest fixes and improvements to the experimental gRPC module.
+- [#3271](https://github.com/grafana/k6/pull/3271),[#3272](https://github.com/grafana/k6/pull/3272) Updates the golangci version and adds the `interfacebloat` linter.
+- [#3279](https://github.com/grafana/k6/pull/3279) Fixes the CI not publishing the SBOM file on a new release.
+- [#3283](https://github.com/grafana/k6/pull/3283) Updates the Go version to 1.21.
```

Comment:
I remember the @mstoykov [was pointing out](https://github.com/grafana/k6/pull/3283#issuecomment-1681742790) that is worth explicitly mentioning this happens in CI. 

Otherwise, it could be interpreted as switching the versions in the k6, like starting to use some functionality from go1.21, which will require a go1.21.

```suggestion
- [#3283](https://github.com/grafana/k6/pull/3283) Updates the Go version in k6's CI to 1.21.
```

Reply:
For me the important part is that k6 is built using `go1.21`, so I would clarify a bit more. What about the following?

```suggestion
- [#3283](https://github.com/grafana/k6/pull/3283) Updates the Go version in k6's CI used to build the binaries. The minimum supported version for extensions continues to be `go1.19`.
```

Reply:
:+1:  for the `build the binaries...` and soft :-1: for the minimum supported version since, in my opinion, the sentence doesn't bring any essential details, but extends the cognitive load of the sentence